src/Image.php

```diff
@@ -1208,9 +1221,9 @@ private function callEnum(
     ) {
         if (self::isImageish($other)) {
             return self::call($base, $this, [$other, $op], $options);
-        } else {
-            return self::call($base . '_const', $this, [$op, $other], $options);
         }
+
+        return self::call($base . '_const', $this, [$op, $other], $options);
```

Comment:
I think the else version is clearer myself, it expresses "use an if() to pick between two returns", whereas without the else it looks more like "if(something) early return; something else". 

But let's go for no else if it's more php-ish.

Reply:
Perhaps this change was not necessary, because the body of the `if` isn't large. However I think this is more php-ish. It's not covered by the PEAR Coding Standards, but it's recommended at the [`best practices`](https://pear.php.net/manual/en/standards.bestpractices.php#standards.bestpractices.returnearly ) (they call it return early, it's also called `unnecessary else` sometimes).

LLVM also recommends this (for C++), see: http://llvm.org/docs/CodingStandards.html#don-t-use-else-after-a-return

Reply:
Those cases are different (imo). The PHP one is early return, which I agree should not need an else. But in php-vips we are not doing early return, instead we are picking between two equally important return paths.  Removing the else makes this (misleadingly) look like the early return pattern, when that's not the case. 

The LLVM one again is different, they want to clarify program flow. Removing the else here does not do that (imo).

Reply:
Ah OK, I agree. Should I revert that particular change?

Reply:
If you have time that'd be great, or I can do it tonight.

Reply:
OK, done: https://github.com/jcupitt/php-vips/commit/6f4be74778fb9b5a37a9a118f1c115e4571020f4

Reply:
Nice!